plugins/spi/SPIOutput.cpp

```diff
@@ -208,12 +206,12 @@ SPIOutput::SPIOutput(const UID &uid, SPIBackendInterface *backend,
   personalities.insert(personalities.begin() + PERS_APA102_COMBINED - 1,
     Personality(APA102_SLOTS_PER_PIXEL,
       "APA102 Combined Control"));
-  personalities.insert(personalities.begin() + PERS_APA102PB_INDIVIDUAL - 1,
-    Personality(m_pixel_count * APA102PB_SLOTS_PER_PIXEL,
-      "APA102 with pixel brightness Individual Control"));
-  personalities.insert(personalities.begin() + PERS_APA102PB_COMBINED - 1,
-    Personality(m_pixel_count * APA102PB_SLOTS_PER_PIXEL,
-      "APA102 with pixel brightness Combined Control"));
+  personalities.insert(personalities.begin() + PERS_APA102_PB_INDIVIDUAL - 1,
+    Personality(m_pixel_count * APA102_PB_SLOTS_PER_PIXEL,
+      "APA102 PB Individual Control"));
```

Comment:
I wonder if something less cryptic might help, or is PB a regularly used term for the APA102? Perhaps "APA102 Pixel Bright Individ Ctrl".

As an aside, if you fancy another little task, we ought to add a static assert or something when the personality string is too long, so we catch it at compile time.

Reply:
i don't think the Pixel Brightness (in the datasheet 'Global Brightness') is regularly used at all....
and i support you in that PB is cryptic ;-)
it would break the naming convention used currently - but i don't know if the 'Control' part is helpfull at all?!

iam open to do the 'static assert thing' but i don't know how to do something like this - can you give me some keywords i can search for?

Reply:
what about 'APA102 Pixel Brightness Individ.' ?

Reply:
Here's an example of a static assert https://github.com/OpenLightingProject/ola/blob/7a96311d03b656fc53b36283d4460774706d3bc3/common/rdm/ResponderHelper.cpp#L112 although @nomis52 may say it's a bad idea or have a better idea.